addons/markers/functions/fnc_initInsertMarker.sqf

```diff
@@ -93,17 +90,9 @@
     _display displayAddEventHandler ["KeyDown", {(_this select 1) in [200, 208]}];
 
     private _hasTimestamp = false;
-    if !((markerText GVAR(editingMarker)) isEqualTo "") then {
+    if !(markerText GVAR(editingMarker) isEqualTo "") then {
```

Comment:
Can `markerText` return anything but a string?
```suggestion
    if !(markerText GVAR(editingMarker) == "") then {
```

Reply:
No, but `isEqualTo` is slightly faster while being exactly as readable. It is used that way all over the code in CBA and ACE.

Reply:
I disagree on the readability (it should have been `===` or something), but this is fine.

Reply:
also a candidate for isNotEqualTo, which will probably land before next ace update